src/ixdat/data_series.py

```diff
@@ -176,7 +176,7 @@ def data(self):
                     f"{self} has {self.N_dimensions} axes but its data is "
                     f"{len(self._data.shape)}-dimensional."
                 )
-        return self._data
+        return self._data.copy()
```

Comment:
If I understand correctly, the `write=False` flag could be a fine solution. Then the .copy() method would only be needed when temporarily manipulation of original data is needed prior to plotting or calculations. 

I encountered the issues with manipulating original data and started using `.copy() ` in the `heat_plot()` method.

Søren saw this implementation and thought it might be a good idea to do globally to ensure data to be immutable.

Reply:
Kenneth's suggestion is better :) 

Can you change the method to the following?
```
    @property
    def data(self):
        """When loading data, Field checks that its dimensions match its # of axes"""
        if self._data is None:
            self._data = self.load_data()
            self._data.setflags(write=False)   # In line with DataSeries being immutable.
            if len(self._data.shape) != self.N_dimensions:
                raise AxisError(
                    f"{self} has {self.N_dimensions} axes but its data is "
                    f"{len(self._data.shape)}-dimensional."
                )
        return self._data
```